build.gradle

```diff
@@ -1,6 +1,14 @@
 group 'multi-int-store'
 version '0.1.0-SNAPSHOT'
 
-repositories {
-    mavenCentral()
-}
\ No newline at end of file
+buildscript {
```

Comment:
Do you know if it makes a difference if `repositories` is inside `buildscript`?

Reply:
I'm actually not sure, I'm not that familiar with gradle, I just copied what I saw.